packages/react/src/theme/theme.tsx

```diff
@@ -1,50 +1,53 @@
-import './styles/theme-dx-dark.scss';
-import './styles/variables-dark.scss';
-import './styles/variables-light.scss';
-import './styles/theme-dx-light.scss';
 import { currentTheme as currentVizTheme, refreshTheme } from 'devextreme/viz/themes';
 import React, { useCallback, useEffect, useMemo, useState } from 'react';
 
 const themeNames = ['dark', 'light'] as const;
-const themeStylesSheets = {};
-const storageKey = 'themeViewer';
+const storageKey = 'app-theme';
+const themePrefix = 'app-theme-';
 
-export type Theme = typeof themeNames[number];
+const prefixes = ['./styles/theme-dx-', './styles/variables-'];
 
-function getCurrentTheme(): Theme {
-  return window.localStorage[storageKey] || 'light';
-}
+const loadStylesImports = async() => {
+  await Promise.all([
+    ...prefixes.flatMap((prefix) => [
+      import(/* webpackChunkName: "app-theme-dark" */ `${prefix}dark.scss`),
```

Comment:
It's better to use themeNames constant everywhere instead of 'dark' and 'light'

Reply:
Ok. I've fixed it in other places, but it this two lines it's impossible, there is a error. I think it because this lines are processed by webpack.